app/src/main/java/org/fossasia/phimpme/opencamera/UI/PopupView.java

```diff
@@ -252,14 +252,24 @@ public void onClick(String option) {
     							toast_message = getResources().getString(R.string.photo_mode_expo_bracketing_full);
     	    				final SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(main_activity);
     						SharedPreferences.Editor editor = sharedPreferences.edit();
+							final String HDRMessage = "preference_photo_mode_hdr";
+							boolean done_dialog = false;
+							if( new_photo_mode == MyApplicationInterface.PhotoMode.HDR && !HDRMessage.equals(sharedPreferences.getString(PreferenceKeys.getPhotoModePreferenceKey(), " "))) {
+								boolean done_hdr_info = sharedPreferences.contains(PreferenceKeys.getHDRInfoPreferenceKey());
+								if( !done_hdr_info ) {
+									showInfoDialog(R.string.photo_mode_hdr, R.string.hdr_info, PreferenceKeys.getHDRInfoPreferenceKey());
+									done_dialog = true;
+								}
+							} else 
+								done_dialog = true;
```

Comment:
Shift it to the line above.

Reply:
thanks for review , i have update the codes